python/lsst/pipe/base/argumentParser.py

```diff
@@ -494,6 +496,12 @@ def parse_args(self, config, args=None, log=None, override=None):
         namespace.config.validate()
         namespace.config.freeze()
 
+        # No environment variable or --output or --rerun specified.
+        if self.requireOutput and namespace.output is None and namespace.rerun is None:
+            self.error("no output directory specified.\n"
+                       "An output directory must be specified with the --output or --rerun\n"
+                       "command-line arguments or the PIPE_OUTPUT_ROOT environment variable.\n")
+
```

Comment:
I suggest removing the bit about PIPE_OUTPUT_ROOT (but see associated request near line 515)